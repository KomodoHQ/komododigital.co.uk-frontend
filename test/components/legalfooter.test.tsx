import React from 'react';
import LegalFooter from '../../src/components/legalfooter';
import renderer from 'react-test-renderer';

test('LegalFooter element is rendered', () => {
  const data = {
  };
  const component = renderer.create(<LegalFooter data={data} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
