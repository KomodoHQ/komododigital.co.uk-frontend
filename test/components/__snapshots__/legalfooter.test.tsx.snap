// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LegalFooter element is rendered 1`] = `
<footer>
  <div>
    Komodo Digital Copyright Â© 2018
    <a
      aria-current="page"
      href="/"
      onClick={[Function]}
    >
      Privacy Policy
    </a>
    |
    <a
      aria-current="page"
      href="/"
      onClick={[Function]}
    >
      Terms and Conditions
    </a>
  </div>
  <div>
    Get in touch on:
    <a
      href="/tel:0191 228 6555"
      onClick={[Function]}
    >
      0191 228 6555
    </a>
  </div>
</footer>
`;
